src/pages/Search/KeywordView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Search } from '@styled-icons/bootstrap/Search';
+import theme from '../../styles/theme';
+
+export const KeywordView = ({ filterResult }) => {
+  return (
+    <Container>
+      {filterResult.map(book => {
+        return (
+          <Wrapper key={book.book_id}>
+            <SearchIcon />
+            <BookMatch>{book.title}</BookMatch>
+          </Wrapper>
+        );
+      })}
+    </Container>
+  );
+};
+
+const SearchIcon = styled(Search)`
+  width: 1rem;
+  margin: 5px;
+  color: ${theme.middleGrey};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.middleGrey};
```
theme provider에서 주입해주고 있으니까 import 하지 말고 사용해주세요